### PR TITLE
- Hardcode enable_metadata_only_fetch_phase_for_inconsistent_updates …

### PR DESCRIPTION
**Pull request by baldersheim**
…and restart_with_fast_update_path_if_all_get_timestamps_are_consistent to true.

- The tests expecting depending on these flags specify these values explicit.

@vekterli PR

### CODE REVIEW

File: storage/src/tests/distributor/distributor_stripe_test.cpp [hunk 1]
```diff
@@ -144,10 +144,8 @@ struct DistributorStripeTest : Test, DistributorStripeTestUtil {
         });
     }
 
-    void configure_update_fast_path_restart_enabled(bool enabled) {
-        configure_stripe_with([&](auto& builder) {
-            builder.restartWithFastUpdatePathIfAllGetTimestampsAreConsistent = enabled;
-        });
+    void configure_update_fast_path_restart_enabled() {
+        configure_stripe_with([&](auto& builder) { (void) builder; });
```
Comment (vekterli) [inline storage/src/tests/distributor/distributor_stripe_test.cpp:148]: Consider adding a small comment on why this is currently a no-op, lest we forget many moons from now

File: storage/src/tests/distributor/distributor_stripe_test.cpp [hunk 2]
```diff
@@ -162,10 +160,8 @@ struct DistributorStripeTest : Test, DistributorStripeTestUtil {
         });
     }
 
-    void configure_metadata_update_phase_enabled(bool enabled) {
-        configure_stripe_with([&](auto& builder) {
-            builder.enableMetadataOnlyFetchPhaseForInconsistentUpdates = enabled;
-        });
+    void configure_metadata_update_phase_enabled() {
+        configure_stripe_with([&](auto& builder) { (void) builder; });
```
Comment (vekterli) [inline storage/src/tests/distributor/distributor_stripe_test.cpp:164]: Consider a comment re: no-op here as well

**Review by vekterli — DISMISSED** [review]
:+1:

File: storage/src/tests/distributor/distributor_stripe_test.cpp (hunk 1 above)
Comment (baldersheim) [reply storage/src/tests/distributor/distributor_stripe_test.cpp]: Good catch, actually unused.

File: storage/src/tests/distributor/distributor_stripe_test.cpp (hunk 2 above)
Comment (baldersheim) [reply storage/src/tests/distributor/distributor_stripe_test.cpp]: Unused, GCed

**Pull request merged**